src/App.tsx

```diff
@@ -1,14 +1,61 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { FilterType } from './types/FilterTypes';
+
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+const getVisibleTodos = (
+  todos: Todo[],
+  sortType: string,
+  query: string,
+): Todo[] => {
+  let visibleTodos: Todo[] = [...todos];
+
+  if (sortType === FilterType.ACTIVE) {
+    visibleTodos = todos.filter(todo => !todo.completed);
+  }
+
+  if (sortType === FilterType.COMPLETED) {
+    visibleTodos = todos.filter(todo => todo.completed);
+  }
+
+  const handleFilterTodos = (value: string, searchQuery: string) => {
+    return (
+      value.toLowerCase().includes(searchQuery.toLowerCase().trim())
+    );
+  };
+
+  return visibleTodos.filter(({ title }) => (
+    handleFilterTodos(title, query)
+
+  ));
```

Comment:
you don't reuse this function anywhere else, not sure if you should separate this code